src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java

```diff
@@ -157,14 +159,29 @@ public abstract class AbstractTranscriptPublisher<T>
         = Logger.getLogger(AbstractTranscriptPublisher.class);
 
     /**
-     * Aspect for successful upload of transcript
+     * A queue used to offload xmpp message sending in a new thread to avoid blocking.
      */
-    private static final String DD_ASPECT_SUCCESS = "upload_success";
+    private static final PacketQueue<Runnable> xmppInvokeQueue = new PacketQueue<>(
```

Comment:
This being static means that order will be preserved for everything. I it makes more sense to have a separate queue for each AbstractTranscriptPublisher